pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -515,11 +753,54 @@ func equalListResponse(respA, respB *podresourcesapi.ListPodResourcesResponse) b
 			if !equalContainerDevices(cntA.Devices, cntB.Devices) {
 				return false
 			}
+
+			if !euqalDynamicResources(cntA.DynamicResources, cntB.DynamicResources) {
```

Comment:
let's fix the typo (cosmetic only, because the typo is used consistently!) before beta. Not blocking now.